core/platform-common/src/main/java/org/sunbird/operations/ActorOperations.java

```diff
@@ -73,6 +73,8 @@ public enum ActorOperations {
   GENERATE_OTP("generateOTP", "OTPCRT"),
   VERIFY_OTP("verifyOTP", "OTPVERFY"),
   SEND_OTP("sendOTP", "OTPNOTI"),
+  GENERATE_OTP_V3("generateOTPV3", "OTPCRT"),
```

Comment:
check the same value can be used or different one required

Reply:
1. GENERATE_OTP_V3 is used to call the generateOTPV3(request) method in the onReceive(Request request) in the OTPActor so this enum would be needed.  
2. Have updated the value of OTPCRT -> OTPCRT3.